tests/StatsdClient.Tests/StatsdBuilderTests.cs

```diff
@@ -71,7 +78,8 @@ public void UDSStatsdServerName()
             Environment.SetEnvironmentVariable(StatsdConfig.DD_AGENT_HOST_ENV_VAR, null);
             Assert.AreEqual("server1", GetUDSStatsdServerName(CreateUDSConfig("server1")));
 
-            Environment.SetEnvironmentVariable(StatsdConfig.DD_AGENT_HOST_ENV_VAR,
+            Environment.SetEnvironmentVariable(
+                StatsdConfig.DD_AGENT_HOST_ENV_VAR,
```

Comment:
I made a comment in #111 about how you renamed one of the constants there to use s different naming convention. I'm not sure why we changed that constant and not others, I just want to mention that in case you think we should rename all constants.

Reply:
The main difference is the visibility. `DD_AGENT_HOST_ENV_VAR` is public and cannot be renamed without a breaking change: https://github.com/DataDog/dogstatsd-csharp-client/pull/111#discussion_r413599394